components/item/MediaCard.vue

```diff
@@ -59,19 +86,22 @@
       europeanaIdentifier: {
         type: String,
         default: ''
+      },
+      isSinglePlayableMedia: {
+        type: Boolean,
+        default: false
       }
     },
 
     data() {
       return {
-        oEmbedData: {},
-        ratio: 56.25
+        oEmbedData: {}
       };
     },
 
     computed: {
       displayImage() {
-        return (this.imageSrc !== '') && !isRichMedia(this.media);
+        return (this.media.thumbnails !== '') && !isRichMedia(this.media);
```

Comment:
we still need to check if the thumbnails attribute exists otherwise plain text would pass as a displayImage if we just checked 
```
displayImage() {
        return (!isRichMedia(this.media);
},
```

so I've switched the first check to confirm the thumbnails attribute exists

```
displayImage() {
        return (has(this.media, 'thumbnails')) && !isRichMedia(this.media);
      },
```